modules/040-node-manager/hooks/internal/conditions/calculate.go

```diff
@@ -221,6 +221,9 @@ func CalculateNodeGroupConditions(
 	}
 
 	isReady := readySchedulableNodes >= minPerAllZone
+	if ng.Type == ngv1.NodeTypeStatic {
+		isReady = readySchedulableNodes == int(ng.Desired)
```

Comment:
we do not have Desired for static

<img width="2288" height="206" alt="Image" src="https://github.com/user-attachments/assets/9485a686-8bd6-468d-8e79-1561bcdf3a67" />